nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -33,6 +33,12 @@
      PHP 7.1 is no longer supported due to upstream not supporting this version for the entire lifecycle of the 19.09 release.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The installer now uses a less privileged <literal>nixos</literal> user whereas before we logged in as root.
```

Comment:
```suggestion
     The installer now uses a less privileged <literal>nixos</literal> user instead of <literal>root</literal>.
```

Reply:
I drop "less privileged" because they're not actually less privileged, and saying it is might lead to questions about well how do I do the thing then.